src/components/Button.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react'
+
+export default class Button extends Component {
+  render() {
+    const { content, type } = this.props
+    
+    return (
+      <button className={`button button-${type}`}>{content}</button>
```

Comment:
Instead of passing `content` as a prop, we can make use of the `children` prop:
```jsx
return (
  <button className=`button button-${type}`}>{this.props.children}</button>
)
```
When we use it:
```jsx
<Button>Content</Button>
```